fmriprep/workflows/anatomical.py

```diff
@@ -17,6 +17,8 @@
 from nipype.interfaces import fsl
 from nipype.pipeline import engine as pe
 
+from ..utils.static_files import get_mni_template
```

Comment:
Thanks! It is because there are a couple of files in data/ that implement exactly the same you have in fmriprep/utils/static_files.py